src/main/java/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import model.Fruit;
+import service.ParserService;
+
+public class ParserServiceImpl implements ParserService {
+    private static final String COMA = ",";
+    private static final int FIRST_POSITION_IN_LINE = 0;
+    private static final int SECOND_POSITION_IN_LINE = 1;
+    private static final int THIRD_POSITION_IN_LINE = 2;
```

Comment:
Let's try to create constant names that explains  business logic meaning e.g.:
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int FRUIT_NAME_INDEX = 1;
    private static final int QUANTITY_INDEX = 2;
```